src/pages/Login/components/GoogleLoginButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+import type { CredentialResponse } from 'google-one-tap';
+import jwt_decode from 'jwt-decode';
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from '../../../core/authentification/auth.hook';
+import { GoogleUser } from '../../../shared/interfaces/google-user';
+import { useLocalStorage } from '../../../shared/hooks/local-storage.hook';
+
+export default function GoogleLoginButton() {
+  const navigate = useNavigate();
+  const { login } = useAuth();
+
+  useEffect(() => {
```

Comment:
We should probably add a [dependency array](https://react.dev/reference/react/useEffect#specifying-reactive-dependencies) for this `useEffect`, as otherwise it would try to initialize after each render.